### ISSUE TITLE
Model ill defined. Code is broken everywhere.

### ISSUE THREAD

**Issue by prohor33** [ShannonAI/mrc-for-flat-nested-ner#14]
There is no layer to predict the probability that start/end should be matched to span, as it's described in original article. Training is done only on start_logits/end_logits.
Loss function defined in a wrong way too, as mentioned in other issues.
Very sorry to say that, but, to be honest, I'm starting to doubt in the results of original article as code do not match to the article and results.

**Comment by stepperL** [comment]
We are currently working on cleaning our codebase and expect to finish in two days.
Once we finish, we will update the repo and let you know.
Many thanks.
